src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/design/innertypelast/Example1.java

```diff
@@ -0,0 +1,30 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="InnerTypeLast"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.design.innertypelast;
+
+// xdoc section -- start
+class Test1 {
+  private String s;
+  class InnerTest1 {}
+  public void test() {} // violation
+  // violation above, method should be declared before inner types.
```

Comment:
We need only one comment about violation , remove `// violation`

Reply:
I changed to single violation comment....
Please review if this is all good